### DISCUSSION TITLE
Welcome to IBAMR Discussions!

### DISCUSSION THREAD

**Discussion by boyceg** [IBAMR/IBAMR#1179] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by drwells** [comment]
@boyceg I'm not opposed to doing this, per se, but I don't see the advantage of this over using the mailing list. Should we keep both?

**Reply by boyceg** [reply]
Discussions is a new GitHub feature. I thought we could try out both. One reason perhaps to prefer Discussions is that it makes it easier to link to issues and PRs. If no one uses it and/or it seems better to stick with the mailing list, we can always disable it.
